src/index.html

```diff
@@ -11,12 +11,104 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Raleway:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__wrap">
+        <a href="/">
+          <img
+            class="header__logo"
+            src="/src/images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__links">
+            <li>
+              <a
+                class="nav__link is-active"
+                href="/"
+              >
+                Apple
+              </a>
+            </li>
+            <li>
```

Comment:
```suggestion
            <li>
              <a
                class="nav__link is-active"
                href="/"
              >
                Apple
              </a>
            </li>
            
            <li>
```

Add an empty line between multiple sibling blocks of html